requirements.txt

```diff
@@ -2,7 +2,6 @@ mpi4py
 numpy
 pytest
 pyvisfile
-pymetis
```

Comment:
Shouldn't we keep this here still, if someone installs mirgecom without using our conda scripts?

Reply:
Fair, added back.